src/app.tsx

```diff
@@ -17,6 +17,8 @@
  */
 
 import React, { FunctionComponent, ReactElement } from "react";
+import { ScriptBasedFlow } from "./components/script-based-flow";
+import { Sidebar } from "./core-components/sidebar";
```

Comment:
Shall we rename `core-components` to `core` and place it inside components(`components/core`).